lib/Gitlab/Api/MergeRequests.php

```diff
@@ -187,12 +187,14 @@ public function showNotes($project_id, $mr_id)
      * @param int $project_id
      * @param int $mr_id
      * @param string $note
+     * @param string $created_at
      * @return mixed
      */
-    public function addNote($project_id, $mr_id, $note)
+    public function addNote($project_id, $mr_id, $note, $created_at = null)
     {
         return $this->post($this->getProjectPath($project_id, 'merge_requests/'.$this->encodePath($mr_id).'/notes'), array(
-            'body' => $note
+            'body' => $note,
+            'craeted_at' => $created_at,
```

Comment:
here's a typo: `craeted_at` should be spelled as `created_at`

please add unit tests and test your changes are actually working.

Reply:
altho I think this deserves better refactoring, see other methods using OptionsResolver and having extra parameters as $parameters array argument